requirements.txt

```diff
@@ -1,3 +1,3 @@
-black==23.3.0
-Pillow==10.0.0
-opencv-python==4.8.0.74
+black
+Pillow
+opencv-python
```

Comment:
Unpinning solves the problem temporarily but ideally, you should've upgraded to whatever version that's supposed to work. 


Also if you're gonna open a PR, put some PR description. In this case, it can be build failure log.

Reply:
Makes sense, Duly noted :)